src/multifile.jl

```diff
@@ -288,10 +301,17 @@ function chunking(v::MFVariable)
     storage,chunksizes = chunking(v1)
 
     if storage == :contiguous
-        return (:chunked, size(v1))
-    else
-        return storage,chunksizes
+        storage = :chunked
+        chunksizes = size(v1)
     end
+
+    if ndims(v) == (length(chunksizes)+1)
+        cat_dim = v.var.dim
+        chunksizes = (chunksizes[1:(cat_dim-1)]...,1,chunksizes[cat_dim:end]...)
+    end
+
+    @assert ndims(v) == length(chunksizes) 
```

Comment:
The `ndims(v) == length(chunksizes) ` has to hold for it to work with `DiskArrays`

Reply:
Possibly a real check and error would be better, `@assert` is not guaranteed to run in future and the error doesn't give much information